schedview/data/sim_metadata.yaml

```diff
@@ -0,0 +1,5 @@
+files:
+  observations:
+    name: 'sample_opsim.db'
+  rewards:
+    name: 'sample_rewards.h5'
```

Comment:
Is this what a sim_metadata file for multiple simulations would look like? I don't see the date of the simulation or any of the extra metadata here, so I'm struggling a bit to understand what the sim archive metadata file really looks like, and how it's used with the updated code in this PR, in particular the parts where directories are searched and the required files are pulled in.

Reply:
Each simulation will have its own sim_metadata file; multiple simulations in a file isn't something that can happen.
This data file is just an absolute minimum needed for running the tests as they are. It will get expanded as new features get added (e.g. searching by date).
What this tells me is that I need to write a documentation page on the archive structure and how I plan for it to be used.

Reply:
Yes, it was just a bit hard figuring it out from the code. There is some information on what is written into the archive, but I also think there were some things we'd talked about previously that have maybe gotten outdated, and also there is a lot of additional information which is being written into the archive. 
Indeed, I find I'm not clear if "archive" is a single directory with a single run or something else.

Reply:
Yes, I've started writing a document on the archive structure. I don't think it needs to be a long document, but still too long to put in a comment, and it applies more generally than just a specific point in the code: someone who wants it shouldn't have to search the code to find it, and it would be bad to have to maintain the documentation in multiple places.
Not sure where the best place to put the document is; in the schedview docs, in the rubin_scheduler docs, and as a stand-alone tech note are all reasonable options, I think. From the point of view of writing it, I don't think it much matters: it's just an rst doc either way. But, it needs to go somewhere.
"Archive" is the directory tree populated with all the simulations and associated files: for each run, there is a directory in the archive with a metadata file and a collection of other files, at a minimum including the sqlite3 database of visits.

Reply:
Any of rubin_scheduler docs, schedview docs, or a tech note seem reasonable to me too. I suppose if the archive is created from rubin-scheduler, perhaps it makes the most sense to put the documentation of it there though (and hopefully having it in the docs and not a tech note means that it will be updated appropriately instead of forgotten).

Reply:
Perhaps an even better reason to put it in the rubin_scheduler docs then, since then it could be completely removed with future updates if needed.